pydjamodb/connection.py

```diff
@@ -127,6 +133,44 @@ def exists_table(self):
         except TableDoesNotExist:
             return False
 
+    def _get_logging_settings(self):
+        logging_settings = settings.PYDJAMODB_DATABASE.get('LOGGING', {})
+        table_settings = logging_settings.get(self.table_name_postfix, {})
+        return table_settings.get('RETURN_CONSUMED_CAPACITY'), table_settings.get('COLUMNS', set())
+
+    def _log_consumed_capacity(self, data, operation_name, updated_columns):
```

Comment:
Is this method used? Maybe I'm missing something but didn't you want to use this method on lines 163-171?

Reply:
this is leftover. I have deleted it. Thanks